library/std/src/lib.rs

```diff
@@ -190,7 +190,7 @@
 #![cfg_attr(not(feature = "restricted-std"), stable(feature = "rust1", since = "1.0.0"))]
 #![cfg_attr(feature = "restricted-std", unstable(feature = "restricted_std", issue = "none"))]
 #![doc(
-    html_root_url = "https://doc.rust-lang.org/nightly/",
+    // NOTE: `html_root_url` is set in src/bootstrap/doc.rs, not here.
```

Comment:
Why is this comment only added for library/std? That seems a bit inconsistent at least. That said, I don't actually have any objection to this.